src/Utils/RMC/RMST.as

```diff
@@ -33,14 +36,12 @@ class RMST : RMS {
     }
 
     void Render() override {
-        if (UI::IsOverlayShown() || PluginSettings::RMC_AlwaysShowBtns) {
+        if (RMC::IsRunning && (UI::IsOverlayShown() || PluginSettings::RMC_AlwaysShowBtns)) {
```

Comment:
I'm not sure what the purpose of "RMC_AlwaysShowBtns" is, but at face value, if it says "always", I obey.

```suggestion
        if ((RMC::IsRunning && UI::IsOverlayShown()) || PluginSettings::RMC_AlwaysShowBtns) {
```

Reply:
What I have here matches the existing logic in `RMS`/`RMC`, I believe. I think it's not technically possible to show buttons without the overlay, since the overlay is where the buttons are rendered, right?